playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/containerized_upgrade.yml

```diff
@@ -1,7 +1,7 @@
 - include_vars: ../../../../../roles/openshift_node/vars/main.yml
 
 - name: Update systemd units
-  include: ../../../../../roles/openshift_node/tasks/systemd_units.yml openshift_version=g_aos_versions.avail_version
+  include: ../../../../../roles/openshift_node/tasks/systemd_units.yml openshift_version=v{{ g_aos_versions.avail_version }}
```

Comment:
@detiber, this is the thing I mentioned to you that was probably never working correctly but was masked by variable precedence previously.